src/_base/harness/attributes/common.yml

```diff
@@ -339,6 +341,10 @@ attributes.default:
     host: redis
     port: 6379
 
+  redis-session:
+    host: = @('redis_session.host')
+    port: = @('redis_session.port')
+
   redis_session:
     host: redis-session
     port: 6379
```

Comment:
why keeping the old reference of redis_session everywhere? for backward compatibility?

Reply:
Yes, in case someone has customised redis_session variants since 0.10.0
Doubtful but... maybe :D

Reply:
It's ok, but it just seems confusing. Should we rather have this in major release, where BC breaks are allowed? so we can then remove the old references.

Reply:
We need a fix for redis-session not being deployed to kubernetes, as it breaks magento 2.
Perhaps we call this 1.0.0?

Reply:
sounds good 👍